### PR TITLE
PMM-7306 Add disable collectors fields.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-7306

### CODE REVIEW

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Mistake

**Review by BupycHuk — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Review by AlekSi — COMMENTED** [review]
What about RDS?

**Comment by askomorokhov** [comment]
> What about RDS?

@AlekSi, Yes, I have asked this question to Roma.  RDS and external exporters are out of scope here. Added to ticket description.

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -71,6 +71,8 @@ message NodeExporter {
   map<string, string> custom_labels = 4;
   // True if exporter uses push metrics mode.
   bool push_metrics_enabled = 7;
+  // List of collector names to exclude from exporter.
```
Comment (AlekSi) [inline api/inventorypb/agents.proto:74]:
```suggestion
  // List of disabled collector names.
```

Nothing "to" there. There same for other "model" messages, but not for requests

File: api/inventorypb/agents.proto [hunk 2]
```diff
@@ -523,6 +533,8 @@ message AddNodeExporterRequest {
   map<string, string> custom_labels = 2;
   // Enables push metrics mode for exporter.
   bool push_metrics = 3;
+  // List of collector names to exclude from exporter.
```
Comment (AlekSi) [inline api/inventorypb/agents.proto:536]:
```suggestion
  // List of collector names to disable in this exporter.
```

The same in other requests

**Pull request merged**